test/models/stage_test.rb

```diff
@@ -141,6 +141,12 @@
         subject.create_deploy(user, {reference: ""})
       end
     end
+
+    it "creates a new deploy with custom command" do
```

Comment:
Test that if we passed a custom command id, then the job has to contain that command.